src/app/ubs-admin/components/ubs-client-profile-page/ubs-client-profile-page.component.scss

```diff
@@ -49,12 +48,20 @@
   }
 
   button {
-    margin: 30px;
+    margin: 30px 0 30px 30px;
     font-family: var(--secondary-font);
   }
 
   h5 {
+    font-weight: bold;
     margin-left: 15px;
+    margin-bottom: 30px;
+    margin-top: 30px;
```

Comment:
Replace this with:
margin: 30px 0 30px 15px;

Reply:
done